airbyte-integrations/connectors/source-todoist/unit_tests/test_streams.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+from http import HTTPStatus
+from unittest.mock import MagicMock
+
+import pytest
+from source_todoist.source import TodoistStream
+
+
+@pytest.fixture
+def patch_base_class(mocker):
+    # Mock abstract methods to enable instantiating abstract class
+    mocker.patch.object(TodoistStream, "path", "v0/example_endpoint")
+    mocker.patch.object(TodoistStream, "primary_key", "test_primary_key")
```

Comment:
You can remove these if you're not using them.

Reply:
done.